evap/staff/views.py

```diff
@@ -1,10 +1,11 @@
 import csv
+from enum import StrEnum
```

Comment:
Starting in Python 3.11 :(

Any other enum is fine for us though, so no big problem